packages/core/src/domain/session/storeStrategies/sessionInLocalStorage.ts

```diff
@@ -34,9 +34,10 @@ function persistInLocalStorage(sessionState: SessionState) {
 
 function retrieveSessionFromLocalStorage(): SessionState {
   const sessionString = localStorage.getItem(SESSION_STORE_KEY)
-  return toSessionState(sessionString)
+  const sessionState = toSessionState(sessionString)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Remove the `const`.